### ISSUE TITLE
RuntimeException: Cannot read from non-readable stream in /var/www/html/vendor/guzzlehttp/psr7/src/Stream.php:208

### ISSUE THREAD

**Issue by Rufflesaurus** [masihyeganeh/YoutubeDownloader#11]
Hi!

I am using this package and it works fine for most of the videos but some specific videos cannot be downloaded and it fails because of the following error:

```
RuntimeException: Cannot read from non-readable stream in /var/www/html/vendor/guzzlehttp/psr7/src/Stream.php:208
```

I tried googling the error but I couldn't find anything.

Any idea what might be the problem? 

Thanks!

**Comment by masihyeganeh** [comment]
Hi
Can you please give me the video ID of those specific videos?

**Comment by Rufflesaurus** [comment]
Omr8iia1L9I is one of them which causes the problem. If you need more of them, I'll dig them out and paste them here.

**Comment by masihyeganeh** [comment]
Thanks for reporting.
I will check to see what is special about these videos. Video [url](http://r19---sn-ab5l6n7s.googlevideo.com/videoplayback?mn=sn-ab5l6n7s&mm=31&id=3a6afc8a26b52fd2&initcwndbps=1231250&itag=22&ip=107.182.226.6&mv=m&mt=1478191137&ratebypass=yes&ms=au&pl=24&lmt=1478153823638914&dur=217.222&source=youtube&key=yt6&mime=video%2Fmp4&sparams=dur%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cnh%2Cpl%2Cratebypass%2Csource%2Cupn%2Cexpire&expire=1478212853&ipbits=0&upn=xu0ELM2bytQ&nh=IgpwcjAzLmxnYTA3Kg03Mi4xNC4yNDIuMjAx) returns error 403 and says `You don't have authorization to view this page.`.
Will fix this in next release soon.

**Comment by masihyeganeh** [comment]
This issue is fixed in version 2.6
Thanks for reporting this problem. please let me know if still there are videos that can't be downloaded.
